go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -80,8 +80,9 @@ func (agent *ActionAgent) Backup(ctx context.Context, concurrency int, logger lo
 
 	// now we can run the backup
 	dir := fmt.Sprintf("%v/%v", tablet.Keyspace, tablet.Shard)
-	name := fmt.Sprintf("%v.%v", time.Now().UTC().Format("2006-01-02.150405"), topoproto.TabletAliasString(tablet.Alias))
-	returnErr := mysqlctl.Backup(ctx, agent.Cnf, agent.MysqlDaemon, l, dir, name, concurrency, agent.hookExtraEnv())
+	backupTime := time.Now().UTC()
```

Comment:
I think the backup time should be pulled from the server's timestamp. Ideally, as of the snapshot time. But I can see someone make an argument for the current time as seen by vttablet. @rafael: thoughts?

Reply:
that is a good point. until now this time was only used to make the backup directory, so it didn't matter, but for purposes of knowing where to replay binlogs from, it should be server time.